[TW-1545] Admin overview refresh (#5309)
postmanlabs/postman-docs: src/pages/docs/administration/admin-overview.md

---
title: "Administration overview"

> updated: 2023-12-19

Reviewer: Bump that up to 01-09

    url: "https://blog.postman.com/managing-team-members-and-roles-in-postman-just-got-easier/"
---

> Postman has a variety of tools you can use to set up, manage, and secure your Postman team. Whether you're setting up a new team or managing an existing one, you can customize your team's experience to align with your organization's goals.

Reviewer 2: ```suggestion
Learn how to set up, manage, and secure your Postman team. Whether you're setting up a new team or managing an existing one, you can customize your team's experience to align with your organization's goals.
```

You must be a [Team Admin or Super Admin](/docs/collaborating-in-postman/roles-and-permissions/#team-roles) to set up, manage, and secure your Postman team.

> If you're setting up a new Postman team, you can check out Postman's [onboarding checklist](/docs/administration/onboarding-checklist/) to walk you through setting up your team.

Author: Thanks! I made updates based on your feedback.

## Contents

* [Set up and manage your Postman team](#set-up-and-manage-your-postman-team)
* [Secure your Postman team](#secure-your-postman-team)
* [Admin FAQs](#admin-faqs)

## Set up and manage your Postman team

> You can [invite and manage collaborators](/docs/administration/managing-your-team/managing-your-team/#managing-team-members) in your Postman team, which includes updating roles assigned to collaborators. You can also [adjust your team's Postman plan](/docs/administration/managing-your-team/managing-your-team/#managing-your-teams-postman-plan) as needed.

Author: I agree. Thanks!

> You can [invite and manage collaborators](/docs/administration/managing-your-team/managing-your-team/#managing-team-members) in your Postman team, which includes updating roles assigned to collaborators. You can also [adjust your team's Postman plan](/docs/administration/managing-your-team/managing-your-team/#managing-your-teams-postman-plan) as needed.

Author: Good idea. I added a note at the end of the section.

You can [configure your team settings](/docs/administration/managing-your-team/team-settings/) to further customize the experience for both your Postman team and your API's consumers. This includes managing your team's public profile and making your team discoverable.

> If you're setting up an [Enterprise team](https://www.postman.com/pricing), you can learn more about the extra features available to you with [Postman Enterprise plans](/docs/administration/enterprise/about-postman-enterprise/). Enterprise features include advanced identity management, reports, a Private API Network, API governance and security controls, and features that enhance collaboration and administrative customization.

Reviewer 2: ```suggestion
If you're setting up an [Enterprise team](https://www.postman.com/pricing), additional features are available to you with [Postman Enterprise plans](/docs/administration/enterprise/about-postman-enterprise/). Enterprise features include advanced identity management, reports, a Private API Network, API governance and security controls, and features that enhance collaboration and administration customization.
```

Author: Thanks! I made updates based on your feedback.

## Secure your Postman team

> Depending on your [plan](https://www.postman.com/pricing), you can secure your Postman team's users by setting up [SSO](/docs/administration/sso/intro-sso/), [SCIM](/docs/administration/scim-provisioning/scim-provisioning-overview/), and [domain verification and capture](/docs/administration/domain-verification-and-capture/domain-capture-overview/). You can also [manage your team's API keys](/docs/administration/managing-your-team/managing-api-keys/), [use the Secret Scanner](/docs/administration/managing-your-team/secret-scanner/) to detect exposed secrets, and [access your team's audit logs](/docs/administration/managing-your-team/audit-logs/).

Reviewer 2: ```suggestion
Depending on your [plan](https://www.postman.com/pricing), you can secure your Postman team's accounts by setting up [SSO](/docs/administration/sso/intro-sso/), [SCIM](/docs/administration/scim-provisioning/scim-provisioning-overview/), and [domain verification and capture](/docs/administration/domain-verification-and-capture/domain-capture-overview/). You can also [manage your team's API keys](/docs/administration/managing-your-team/managing-api-keys/), [use the Secret Scanner](/docs/administration/managing-your-team/secret-scanner/) to detect exposed secrets, and [access your team's audit logs](/docs/administration/managing-your-team/audit-logs/).
```

Reviewer 2: Consider breaking up into bullets for easier scanning.

Author: @nerminamiller-postman I updated the section to use bullet points instead. Please let me know what you think. Thanks!

Reviewer 2: Looks great!

## Admin FAQs

> For frequently asked questions about Postman teams, see [Admin FAQs](/docs/administration/admin-faqs/).

Reviewer 2: ```suggestion
To learn more about managing Postman teams, see [Admin FAQs](/docs/administration/admin-faqs/).
```
